app/assets/javascripts/components/markdown-editor.js

```diff
@@ -26,6 +27,9 @@ MarkdownEditor.prototype.init = function () {
   // Handle events
   this.$previewButton.addEventListener('click', $module.boundPreviewButtonClick)
   this.$editButton.addEventListener('click', $module.boundEditButtonClick)
+
+  // Bubble focus events
+  this.bubbleFocus(this.$input, this.$container)
```

Comment:
Could you rename this to something like `toggleContainerFocus`, as it made me think it was a generic method?

Reply:
yep, is not as generic as the names suggests, but it's not a toggle thing either (as it can receive consecutive focus/blur events without alternating). `reflectElementFocus`?